AGPs/Edgeware Lockdrop Proposal for Aragon Association.md

```diff
@@ -0,0 +1,80 @@
+---
+AGP:  TBD
+Title:  Edgeware Lockdrop Participation
+Author: Thom Ivy (@ThomIvy)
+Status: Stage III
+Track: Finance
+Created: 2019-04-04
+---
+
+# Edgeware Lockdrop Proposal for Aragon
+A proposal to transfer funds from the Aragon multisig to a lockdrop contract for the purpose of participating in the Edgeware network launch.
+
+## Address of the Transfer Recipient:
+TBD, to be created at the time of lockdrop participation.
```

Comment:
I would normally disapprove of leaving the address unspecified, but since there is a link to the lockdrop contract code at the bottom of the proposal, and the proposal makes clear that the transfer should be made to a lockdrop contract under the control of the Aragon Association I think it is clear enough what needs to happen for this proposal to be fulfilled.

Reply:
@stefanobernardi this seems like one of those things similar to the Community Funding DAO where the Association is expected to do something (there it was create a DAO and fund it, here it is deploy a contract and lock up some tokens), per my comment above the proposal seems acceptable to me from an AGP editor standpoint but do you have any feedback to give before this moves on to the AA board review stage?